Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -36,9 +36,6 @@ function spawn_model() {
 
 	gz model --spawn-file=/tmp/${MODEL}_${N}.sdf --model-name=${MODEL}_${N} -x 0.0 -y $((3*${N})) -z 0.0
 
-    #For RTPS, run micrortps_agent like below
```

Comment:
Note this is necessary, but should live in the PX4 Devguide.

Reply:
Yes, I will write it in PX4 Devguide. Thanks.